versions-maven-plugin/src/main/java/org/codehaus/mojo/versions/UseLatestVersionsMojo.java

```diff
@@ -85,6 +86,13 @@ public class UseLatestVersionsMojo extends UseLatestVersionsMojoBase {
     @Parameter(property = "allowSnapshots", defaultValue = "false")
     protected boolean allowSnapshots;
 
+    /**
+     * Whether to allow pre-release versions (alpha, beta, milestone, rc) when searching
+     * for the latest version of an artifact.
+     */
+    @Parameter(property = "allowPreReleases", defaultValue = "false")
```

Comment:
There is a new parameter ....
What `since` tag should have?

@andrzejj0

Reply:
@since 2.20.0 

(feature release)